### ISSUE TITLE
Crystal Sky - android mobile sdk - integration

### ISSUE THREAD

**Issue by Djacob502** [dji-sdk/Mobile-SDK-Android#201]
This is a request to make it easier to switch between a custom DJI app and the DJI GO 4 on the crystal sky. Switching between apps by going into the default setting and clearing the defaults is a pain point for clients. 
At least on the Crystal Sky where DJI controls the android OS - could a method be created to make this process seemless? For instance, if the DJI GO 4 app is opened, then that app is considered the default and connects to the drone. If a custom app is then opened, to have that app considered the default and connect with the drone. 
Next, when the DJI GO 4 app is opened, to automatically remove the custom app default and instead make the switch back to the DJI GO 4 app seemless without any additional user input. 
Not having clients go into the android settings to stop apps and remove defaults would be a nice feature.

**Comment by nextfullstorm** [comment]
As a workaround you may start DJI GO or your app ticking "just once", not "always". This way you can select app EVERY time hardware connects. Not the best solution, but no need to clear defaults every time.

**Comment by jtuchek** [comment]
Custom apps are supported on the Crystal Sky now?

**Comment by Djacob502** [comment]
Yes - custom apps are now allowed on crystal sky. However once we put our app on crystal sky - DJI will not support the custom app on cs at this time. Also - google play framework is not supported so we are rewriting our app to use mapbox instead of google maps to provide a mapping solution.  In addition it is necessary to slide load the apps and not download the apps via google play. Given the restrictions it looks like the 2 custom apps I've placed on cs work well. It is nice the cs recognizes the app and connects to the drone without error which has been a large problem for us in the past.
